slither/core/cfg/node.py

```diff
@@ -230,6 +230,10 @@ def __init__(
         self.scope: Union["Scope", "Function"] = scope
         self.file_scope: "FileScope" = file_scope
 
+        # Record the continue destination in the STARTLOOP node, so `continue` node can
+        # find the correct jump destination.
+        self.continue_destination: Node = None
```

Comment:
I think the type should be `Optional[Node]`.

As a side note, sum types would be useful here. Then the STARTLOOP nodetype could of a `continue_destination` field of type `Node` and other node types wouldn't have a `continue_destination` field.